src/components/common/dateTimeInterval/DateTimeInterval.tsx

```diff
@@ -2,29 +2,46 @@ import React from "react";
 import PropTypes from "prop-types";
 import { DateTimeField } from "@madie/madie-design-system/dist/react";
 import { FormControl } from "@mui/material";
+import { DateTimePicker } from "@mui/x-date-pickers/DateTimePicker";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
 
 interface DateTimeIntervalProps {
   label: string;
   startDateTime: object;
   handleStartDateTimeChange: Function;
   endDateTime: object;
   handleEndDateTimeChange: Function;
+  canEdit: boolean;
 }
 
+export const dateTimeTextFieldStyle = {
+  fontFamily: "Rubik",
+  fontStyle: "normal",
+  fontWeight: "500",
+  fontSize: "16px",
+  lineHeight: "19px",
+  height: "19px",
+  color: "#125496",
+};
+
 const DateTimeInterval = ({
   label,
   startDateTime,
   handleStartDateTimeChange,
   endDateTime,
   handleEndDateTimeChange,
+  canEdit,
 }: DateTimeIntervalProps) => {
   return (
     <div>
+      <h5 style={dateTimeTextFieldStyle}>Time Range</h5>
       <FormControl
-        style={{ display: "flex", gap: "40px", flexDirection: "row" }}
+        style={{ display: "flex", gap: "32px", flexDirection: "row" }}
```

Comment:
Can we use tailwind here? the only reason is that Tailwind Classes are responsive and will reduce overhead in the future

Reply:
I tried with tailwind but I didn't workout. I used tw="flex flex-row gap-8" in the form control but it is not placing the divs side by side

Reply:
fixed it, thanks for the help